docs/getting-started/invite-members/README.md

```diff
@@ -64,13 +65,13 @@ Type in the email address of the new member. If OpenProject does not find an exi
 
 An email invitation will be sent out to the user with a link to [create an account](../sign-in-registration/#create-a-new-account) for OpenProject.
 
-![invite-new-members](1566224961670.png)
+![invite-members-via-email](Bildschirmfoto 2022-03-02 um 14.19.56.png)
```

Comment:
please rename the image file and change the name here, so there are no spaces in it.